clients/acm.d.ts

```diff
@@ -0,0 +1,350 @@
+import {Request} from '../lib/request';
+import {Response} from '../lib/response';
+import {AWSError} from '../lib/error';
+import {Service} from '../lib/service';
+declare class ACM extends Service {
+  /**
```

Comment:
Also, as a modification to what I said above, instead of merging together all of the input shapes, it would be better if each potential param is explicitly listed, since they are all optional here, and we don't want to keep the requirements of the input params. Most services reuse most of their input params, so for many services the list of possible params may actually be shorter than the list of input shapes. So instead of what I have above, it could be more like:
```
interface ServiceParams {
    params?: ACM.AllInputParams
}
```
and inside the `ACM` namespace have:
```
interface AllInputParams {
    CertificateArn?: Arn;
    Tags?: TagList;
    CertificateStatuses?: CertificateStatuses;
    ...
}
```